components/Auth/vregister.vue

```diff
@@ -422,4 +454,7 @@ text-align: center; font-family: iran-yekan; font-size: 1.4em; outline:none; bor
         margin-bottom 16px
         color #fd3259
 
+    .term-of-use
```

Comment:
```suggestion
    .terms-of-use
```
please update the other places accordingly @behroozmirzaie7